clang/lib/Sema/SemaDeclCXX.cpp

```diff
@@ -951,28 +959,124 @@ Sema::ActOnDecompositionDeclarator(Scope *S, Declarator &D,
   return New;
 }
 
+// CheckBindingsCount
```

Comment:
We [don't start function comments with the function name](https://llvm.org/docs/CodingStandards.html#commenting:~:text=Don%E2%80%99t%20duplicate%20function%20or%20class%20name%20at%20the%20beginning%20of%20the%20comment.). (We used to but that's an old style we're phasing out.)